src/client/atoms.ts

```diff
@@ -0,0 +1,26 @@
+import { Prisma } from '@prisma/client'
+import { atom } from 'jotai'
```

Comment:
I prefer to name a file like this as: `app.state.js` -> so grouping atoms by type of state

I might have `ui.state.js` and `data.state.js` to group atoms